lib/vagrant-service-manager/plugin.rb

```diff
@@ -19,6 +19,7 @@ class Plugin < Vagrant.plugin('2')
 
       action_hook(:servicemanager, :machine_action_up) do |hook|
         hook.append(Service::Docker)
+        hook.append(Service::OpenShift)
```

Comment:
This needs to be if-def'ed off so that OpenShift is default for RH CDK only.  We don't need hte docker append if we do OpenShift, afaik.